src/Tasks/Microsoft.NET.Build.Tasks/sdk/Sdk.props

```diff
@@ -32,6 +32,59 @@ Copyright (c) .NET Foundation. All rights reserved.
     <UsingNETSdkDefaults>true</UsingNETSdkDefaults>
   </PropertyGroup>
 
+
+  
+  <!-- We need to put the UseArtifactsOutput logic after the import of Directory.Build.props, but before the MSBuild Project Extensions .props import.
+       However, both of these things happen in Microsoft.Common.props with no opportunity to insert logic in between them.
+
+       So what we do here is duplicate the Directory.Build.props import logic from Microsoft.Common.props, and then set ImportDirectoryBuildProps to
+       false so that it doesn't get imported twice.
+
+       Alternatively, we could add a hook in MSBuild to define a file to import at the right location, to avoid duplicating the logic here.
+       -->
+  <PropertyGroup>
+    <ImportDirectoryBuildProps Condition="'$(ImportDirectoryBuildProps)' == ''">true</ImportDirectoryBuildProps>
```

Comment:
When we communicate this feature to users we should be very clear they shouldn't add this to their project files.

Reply:
This is implementation that users generally won't see.  The ImportDirectoryBuildProps property is something that MSBuild has supported since (I believe) the support for Directory.Build.props was added.